casper/src/test/scala/coop/rchain/casper/helper/RhoSpec.scala

```diff
@@ -32,7 +32,7 @@ object RhoSpec {
       testResultCollector: TestResultCollector[F]
   ): Seq[SystemProcess.Definition[F]] = {
     val testResultCollectorService =
-      Seq((5, "assertAck", 25), (1, "testSuiteCompleted", 26))
+      Seq((5, "assertAck", 26), (1, "testSuiteCompleted", 27))
```

Comment:
I assume you had to change these because of the conflicts with the production system processes. If that's the case I think it was discussed in the standup that we should use a higher number. What about if the test ones start at 1000?

Reply:
Bytes only go until 127 but sounds good